app-tests/cra-button/src/App.js

```diff
@@ -2,39 +2,12 @@ import React from 'react';
 import './App.css';
 import { Button } from '@espressive/cascara';
 
-import {
```

Comment:
Hi @brianespinosa. After removing 'semantic-ui-react' from the root package.json, this file was not getting the reference. Considering that this is just for testing purposes and the component was to try the button, I kept that in place and remove the references for SUIR.

Reply:
Hi team, I think removing the `semantic-ui-react` dependency is the way to go, since it is not used in Cascara anymore, it is not required in this demo app. @brianespinosa what do you think?

Reply:
@rociobaena that page would not get the reference because in this PR SUIR was removed as a dependency of that test app. We do not need to remove SUIR from the test app as those are not part of the monorepo install process. These test apps are basically in isolation. We can keep the button here.

The point behind this specific test app was to see what our Cascara button looks like along the old SUIR button, and in an environment where the legacy CSS is also present. That is why we see both buttons. This is a case where we should leave it. Again, this is a separate test app and the dependency does not get installed unless someone is specifically in this test app. cc @lordpixel

Reply:
File recovered